README.md

```diff
@@ -1,8 +1,14 @@
-This is the Github-hosted branch of edx-platform that is behind
-http://docs.edx.org
+## Overview
+This is the Github-hosted and tied to the master branch that is behind
+http://docs.edx.org. 
 
+## Search
 
-The search.html page is powered by cse.google.com, configured here:
-https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a
+The `search.html` page is powered by Google Programmable Search Engine and can be configured [here](https://cse.google.com/cse/setup/basic?cx=003935656615667250026:jtodz7s8c9a):
 
-The current list of sites that should be configured there is in search-sites.txt.
+The current list of sites that should be configured there is in `search-sites.txt`.
+
+## Contribution
```

Comment:
I like the idea, but this should probably be in `.github/CONTRIBUTORS.md` (see some other edX repos for examples). This is a more standard and GitHub-integrated way of surfacing that info.

Reply:
I think github talks about having that file at the root, not in `.github/` can you confirm? 
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Reply:
^ The doc allows for that file in root, `docs`, or `.github`. `.github` is my personal preference but it could be in any of those 3.

Reply:
yes, right :)